hasura.planx.uk/migrations/1709563913656_create_public_view_applications_summary/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE VIEW "public"."applications_summary" AS
+SELECT
+  s.session_id,
+  s.team_slug,
+  s.service_slug,
+  submitted_at,
+  user_invited_to_pay,
+  COALESCE(ps.status, 'N/A') AS payment_status,
+  COALESCE(ps.amount, 0) AS amount,
+  COALESCE(ps.created_at::text, 'N/A') as  payment_date,
+  sent_to_bops,
+  sent_to_uniform,
+  sent_to_email
+FROM submission_services_summary s
+LEFT JOIN (
+  SELECT DISTINCT ON (session_id) *
+  FROM payment_status
+  ORDER BY session_id, created_at DESC
+) ps ON ps.session_id = s.session_id
+WHERE submitted_at IS NOT NULL;
```

Comment:
A couple questions about this view: 
- I'd expect to see GOV UK ID, BOPS ID and Uniform IDs exposed for parity with #planx-notifications rather than a yes/no "sent_to". I think councils currently have a high need for a simple lookup from Planx ID <> integrations ID, rather than needing specific application details like the payment amount exposed here if that makes sense
- It feels very hard to semantically differentiate between `applications_summary` and `submission_services_summary` - is there a reason to not simply update `submission_services_summary` directly to include these additional columns?

Reply:
Data Exposed: 
- That's a very good point regarding GOV PAY ID, BOPS ID and Uniform IDs.
- I hadn't properly parsed #planx-notifications so was just seeing what I thought were PlanX session ids 🤦 
- I'll get this updated 👍 
- Do we still want payment details other than the id, I think it was mentioned in the ticket but I got the wrong end of the stick?

Reply:
View:

- I had originally used `submission_services_summary` but as that's exposed via Metabase and in my understanding would have the potential to be more public facing I thought I better hide the extra info like payment details
- I hadn't added it but I thought there might be a chance we'd add more explicit PII like email or name to this view depending on how CO would use it?
- If that's not the case and the data we'd have here wouldn't be sensitive then I think it could be added to the existing view 👍

Reply:
I'd vote we aim for feature parity with the information in #planx-notifications which is not currently PII and would therefore let us use the same view. 

As I understand it this is meant to sort of be more a receipt of pay & send events, and it's the back office systems responsibility to display actual application _details_. If further requests come from POs, we could of course re-evaluate.

What I think the data we want is:
- session id 
- `payment_requests.status`, `govpay_payment_id`, `created_at` (amount not necessary)
- `bops_applications.bops_id` (maybe `destination_url` to differentiate between v1 & v2 recently)
- `uniform_applications.idox_submission_id` (`submission_reference` = session id so can be omitted)
- `email_applications.recipient` 
- submission timestamp for each applicable destination

That would all be considered non-PII as it's just how various identifiers link up. 

There's one consideration with my proposal though that I'm not sure your original data structure accounted for: there will be many `payment_requests` & `bops_applications` per each session and I think we want to show each of them, not just the latest. This might make a case for a separate view or just a note to `group by session_id` if able to use exisitng `submission_services_summary`?

Does that help at all?

Reply:
This is super helpful thanks Jess!

I'll have a look into adding this to the view and I could maybe create arrays of `payment_requests` and `bops_applications` to maintain the expectation of one row in `submission_services_summary` per `session_id` 🤔

Reply:
Arrays make sense if easy enough to work with on the frontend - I'd imagine GraphQL should be good for this :+1: 

And just to be totally clear - any pay or send destination can technically be many events to one session id because of re-submissions, but payments & BOPS are just the two that happen most often/intentionally !

Reply:
I've added a PR to update the `submission_services_summary` view to hopefully expose the data as discussed: https://github.com/theopensystemslab/planx-new/pull/2879